### DISCUSSION TITLE
How create a mutation in a relational table after User Sign Up?

### DISCUSSION THREAD

**Discussion by ribantech** [nhost/nhost#1857] [General]
My sign up form includes name, email & org name fields. I want to insert the Org details in the Org table with a FK as User id. Now I have also kept email should be verified as checked. I need help to set up this workflow. 

signUpEmailPassword does not return the User id if needsEmailVerification is true. So, after User email verification, how can I retrieve the Org Name to create a mutation?

**Comment by dbarrosop** [comment]
It is going to depend a bit on your exact use case but I'd suggest looking into using a postgres trigger, that'd allow you to run any arbitraty SQL on row insert, for instance.
